unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/WorldChatWindowController.cs

```diff
@@ -105,7 +105,6 @@ public void SetVisibility(bool visible)
 
     private void HandleUnfriend(string friendId)
     {
-        friendsController.RemoveFriend(friendId);
```

Comment:
could you remove the method?

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2346/commits/50313cae1cca9da2fd00baacc8ea3badbf35d1a2.